src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -340,4 +360,22 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 		}
 		return undefined;
 	}
+
+	public fileToBase64 = (filepath, getType) => {
```

Comment:
Also, we already have a function that decodes a base64 image : https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/charts/browser/actions.ts

This uses the built-in [atob function](https://developer.mozilla.org/en-US/docs/Glossary/Base64) which seems like a  much simpler approach - so for this you'd just fetch the contents of the file using the file service and then encode it using btoa.